src/cljs/ataru/virkailija/application/view.cljs

```diff
@@ -345,7 +345,9 @@
                :on-click (fn []
                            (select-application (:key application))
                            (dispatch [:state-update #(assoc-in % [:application :selected-review-hakukohde] hakukohde-oid)]))}
-              (from-multi-lang (:name hakukohde))]
+              (from-multi-lang (:name hakukohde))
+              (when (contains? hakukohde :tarjoaja-name)
```

Comment:
Näitä ei enää tarvita kun tekee `:tarjoaja-name` kentästä pakollisen hakukohteelle.

Reply:
:+1: